components/Header/HeaderItems.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { usePathname } from 'next/navigation';
+import { LINKS } from '@/config/consts';
+import { NavDropdown } from './NavDropdown';
+import Link from 'next/link';
+
+interface HeaderItemsProps {
+  headerText: string[];
+}
+
+const HeaderItems = ({headerText}: any) => {
+  const pathname = usePathname();
```

Comment:
Please don't use `any` for the type.
Also, you can use restructuring for the component props